lab-Marshall/lib/cowsay.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const moo = require('cowsay');
+
+const cowsay = module.exports = function(a) {
+  let newText = process.argv[2] || a;
```

Comment:
You don't need to use 'let' here as any variable you declare here will already be scoped to your function. 'let' is more appropriate for block scoping where the block is not a function, like with 'if' statements or 'for' loops.